TwoPaneLayout/library/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="android">
```

Comment:
I think you have to change the package name used here, so the app's resources (R) file uses the library package (https://developer.android.com/guide/topics/manifest/manifest-element)